packages/client/hmi-client/src/components/models/tera-model-configuration.vue

```diff
@@ -182,13 +182,13 @@ const selectedInitials = ref<string[]>([]);
 const selectedParameters = ref<string[]>([]);
 
 const configurations = computed<any[]>(
-	() => editableModelConfigs.value?.map((m) => m.configuration.semantics.ode) ?? []
+	() => editableModelConfigs.value?.map((m) => m.amrConfiguration.semantics.ode) ?? []
 );
 
 // TODO: Clean this up and use appropriate loops
 const modelConfigurationTable = computed(() => {
 	if (editableModelConfigs.value && !isEmpty(configurations.value)) {
-		console.log('Configuration', configurations.value);
+		// console.log('Configuration', configurations.value);
```

Comment:
Why comment this out without removing?

Reply:
This component is still WIP I debug a lot here. It'll be ready for clean up later